<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QImage2MatDialog</class>
 <widget class="QDialog" name="QImage2MatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QImage2MatDialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_imagePath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_qimg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_mat">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>QImage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>610</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mat to QImage</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
